docs/prerequisites.md

```diff
@@ -0,0 +1,44 @@
+# Prerequisites
+
+## thundernetes foundational knowledge. 💖
+
+Here you will find resources that will fill the knowledge gaps when working with technologies within thundernetes. 
+
+👉**thundernetes** was named after a combination of the words *thunderhead* and *kubernetes*. 
+Thunderhead is the internal code name for the [Azure PlayFab Multiplayer Servers](https://azure.microsoft.com/services/playfab/multiplayer-services/) service. 
+
+## Docker and Containerisation 🚢
+- [Docker Basics](https://www.docker.com/101-tutorial)
+
+## Kubernetes 🌼
+Kubernetes is built on top of Docker to run containers at scale across many machines. We recommend that you have knowledge in the following Kubernetes areas before getting started with thundernetes.
+
+### Kubernetes Resources 
+- [Kubernetes Basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/)
+- [Kubernetes core concepts for Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/concepts-clusters-workloads)
+- [kubectl](https://github.com/virtual-kubelet/virtual-kubelet)
+- [kind](https://kind.sigs.k8s.io/)
+  -   kind is a tool for running local Kubernetes clusters using Docker container “nodes”.
+kind was primarily designed for testing Kubernetes itself, but may be used for local development or CI.
+
+## Game Servers 👾
+Thundernetes is an preview project from the Azure PlayFab Multiplayer Servers (MPS) team that enables you to run Linux game servers that use the PlayFab Game Server SDK (GSDK) on your Kubernetes cluster. 
+
+- [Integrating Game Servers with Game Server SDK (GSDK)](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/integrating-game-servers-with-gsdk)
+- [Azure PlayFab Multiplayer Servers Gameserver SDK (GSDK)](https://github.com/PlayFab/MpsSamples)
```

Comment:
that's the GSDK samples repo, we can keep it. The actual GSDK repo is [https://github.com/PlayFab/gsdk](https://github.com/PlayFab/gsdk)